server/data/reportingClient.test.js

```diff
@@ -1,6 +1,7 @@
 const moment = require('moment')
 
 const reportingClient = require('./reportingClient')
+/** @type {any} */
```

Comment:
its a bit of an issue using jest.mocks as jest makes all owned functions mocks but typescript isn't aware of that.
When using typescript proper, generally have to use casting here